engine/platform/android/dlsym-weak.cpp

```diff
@@ -32,63 +32,71 @@
 #include <dlfcn.h>
 #include "linker.h"
 
-static Elf_Sym* soinfo_elf_lookup(soinfo* si, unsigned hash, const char* name) {
+static Elf_Sym* soinfo_elf_lookup( soinfo* si, unsigned hash, const char* name )
+{
     Elf_Sym* symtab = si->symtab;
     const char* strtab = si->strtab;
 
 	if( si->nbucket == 0 )
 		return NULL;
 
-    for (unsigned n = si->bucket[hash % si->nbucket]; n != 0; n = si->chain[n]) {
+	for( unsigned n = si->bucket[hash % si->nbucket]; n != 0; n = si->chain[n] )
+	{
         Elf_Sym* s = symtab + n;
```

Comment:
spaces mixed with tabs

Reply:
I don't think it should be fixed right now. 

The original file used spaces and nobody noticed. We can convert it after merge.